part-iv-advanced-tuning/metrics-and-monitoring.md

```diff
@@ -111,7 +111,7 @@ public class MonitoringConfiguration {
 ### Interceptor Logging
 
 Another good approach to track the flow of messages throughout an Axon application is by setting up the right interceptors in your application.  
-As you might now there are two flavors of interceptors, the Dispatch and Handler interceptors (like discussed [here](../part-iii-infrastructure-components/command-dispatching.md#command-interceptors) for commands and [here](../part-iii-infrastructure-components/query-processing.md#handler-interceptors) for queries), which intercept a message prior to publishing (Dispatch Interceptor) or whilst it is being handled (Handler Interceptor).
+As you might now there are two flavors of interceptors, the Dispatch and Handler interceptors (like discussed [here](../part-iii-infrastructure-components/command-dispatching.md#command-interceptors) for commands, [here](../part-iii-infrastructure-components/event-processing.md#event-interceptors) for events and [here](../part-iii-infrastructure-components/query-processing.md#handler-interceptors) for queries), which intercept a message prior to publishing (Dispatch Interceptor) or whilst it is being handled (Handler Interceptor).
```

Comment:
It's a reference guide. Let's not make assumptions on what the reader knows ;-)
Suggestion: "There are two types of interceptors [...]"

Reply:
Daaawh, fair enough, will adjust.

Reply:
Removed the assumption the reader might be aware of the fact there are two interceptor formats.